features/conversation/dm-conversation.screen.tsx

```
@@ -0,0 +1,290 @@
+          return;
+        }
+
+        if (remoteAttachment) {
+          await conversation.send({
+            remoteAttachment,
+          });
+        }
+
+        if (text) {
+          await conversation.send(text);
+        }
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)